src/assets/scripts/modules/download-filter/download-filter.js

```diff
@@ -0,0 +1,59 @@
+import List from 'list.js';
+
+const component = document.querySelector('[data-downloads]');
+const filter = document.querySelector('[data-downloads-filter]');
+const sort = document.querySelector('[data-downloads-sort]');
+const formatButtons = Array.from(document.querySelectorAll('[data-downloads-format]'));
+
+function initDownloadFilter(formats) {
```

Comment:
Now you inject `formats` but it's not used anywhere

Reply:
removed that param